### ISSUE TITLE
upcoming features and mini roadmap

### ISSUE THREAD

**Issue by atmonshi** [lara-zeus/sky#46]
hello everyone
here is what I am planning to add to `Sky`.
any requests or feedback are welcome 🙏🏽

> I will update this regularly.

- [ ] support other editors like markdown. and can be set from the config
- [x] enable the status `private` abort if not the author #74 
- [ ] show the form if the password is set.
- [ ] enable the status `future` by adding a job to publish the posts
- [ ] more themes
- [x] allow the author to preview the post or the page regardless of the status #74 
- [ ] add authors resources, and permissions using the default users or a new table.
- [ ] add tests
- [x] set default featured image from config #73
- [x] set featured image by URL #72

**Comment by ariaieboy** [comment]
hi
I didn't use your package yet but I plan to use it for my personal blog very soon.

one cool feature could be a share to socials button.

it can be as simple as copying a short text + linking of the post so we can share it on socials like Twitter

**Comment by Nico-Stark** [comment]
Hi,
i use your package now, and will take a look about it.
I just noticed something that I was able to solve quickly :D .
There is a language switcher on the post resource.
I could only choose English or Arabic.
Maybe it would be good if you do the following in zeus-sky.php config:

`'translatable_Locales' => (config('app.available_locales') !== null) ? array_values(config('app.available_locales')) : ['en','ar'],`

Since I like to manage global things in one place, that might be better.
Of course you would then only have to:

`'available_locales' => [
        'Arabic' => 'ar',
        'English' => 'en',
...
    ],`

enter in the app.php config
